db/db_flush_test.cc

```diff
@@ -290,6 +290,38 @@ TEST_F(DBFlushTest, ManualFlushFailsInReadOnlyMode) {
   Close();
 }
 
+TEST_F(DBFlushTest, CFDropRaceWithWaitForFlushMemTables) {
+  Options options = CurrentOptions();
+  options.create_if_missing = true;
+  CreateAndReopenWithCF({"pikachu"}, options);
+  SyncPoint::GetInstance()->DisableProcessing();
```

Comment:
This line should go at the end of the test. Every EnableProcessing should be paired with a DisableProcessing. 
Same comment for the other test.

Reply:
Looks like the safest way is to have `DisableProcessing` both at the beginning and in the end, because not all our tests follow the rule you just mentioned. https://github.com/facebook/rocksdb/blob/master/db/db_test.cc

Reply:
😮 !!